pyoptsparse/pySLSQP/pySLSQP.py

```diff
@@ -257,7 +257,7 @@ def slgrad(m, me, la, n, f, g, df, dg, x):
             self.optProb.comm.bcast(-1, root=0)
 
             # Store Results
-            inform = np.asscalar(mode)
+            inform = mode.item()
```

Comment:
What was the issue here that required the change? Did this mean all the informs were broken previously?

Reply:
Nothing was broken, but the tests were printing deprecation warnings.

Reply:
I see this is the `asscalar` deprecation warning.